### PR TITLE
DBZ-2980 Additional logging for number and type of sql operations

### PR DESCRIPTION
**Pull request by mpermar**
Adding extra logs to make explicit the type of DML operations

### CODE REVIEW

File: debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/LogMinerQueryResultProcessor.java [hunk 1]
```diff
@@ -159,6 +159,11 @@ int processResult(ResultSet resultSet) {
             if (operationCode == RowMapper.INSERT || operationCode == RowMapper.DELETE || operationCode == RowMapper.UPDATE) {
                 LOGGER.trace("DML,  {}, sql {}", logMessage, redoSql);
                 dmlCounter++;
+                switch(operationCode) {
+                    case RowMapper.INSERT: insertCounter++; break; 
+                    case RowMapper.UPDATE: insertCounter++; break;
```
Comment (Naros) [inline debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/LogMinerQueryResultProcessor.java:164]:
```suggestion
                    case RowMapper.UPDATE: updateCounter++; break; 
```

File: debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/LogMinerQueryResultProcessor.java [hunk 2]
```diff
@@ -159,6 +159,11 @@ int processResult(ResultSet resultSet) {
             if (operationCode == RowMapper.INSERT || operationCode == RowMapper.DELETE || operationCode == RowMapper.UPDATE) {
                 LOGGER.trace("DML,  {}, sql {}", logMessage, redoSql);
                 dmlCounter++;
+                switch(operationCode) {
+                    case RowMapper.INSERT: insertCounter++; break; 
+                    case RowMapper.UPDATE: insertCounter++; break; 
+                    case RowMapper.DELETE: insertCounter++; break;
```
Comment (Naros) [inline debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/LogMinerQueryResultProcessor.java:165]:
```suggestion
                    case RowMapper.DELETE: deleteCounter++; break; 
```

**Review by Naros — CHANGES_REQUESTED** [review]
I've suggested some very minor changes, otherwise LGTM.

**Review by Naros — APPROVED** [review]
I've applied my suggested changes to the PR; this LGTM.

**Comment by mpermar** [comment]
Oh 🤦‍♂️😅Thanks @Naros !

**Pull request merged**